src/BinDeps.jl

```diff
@@ -6,8 +6,8 @@ using Compat
 
 export @make_run, @build_steps, find_library, download_cmd, unpack_cmd,
     Choice, Choices, CCompile, FileDownloader, FileRule,
-    ChangeDirectory, FileDownloader, FileUnpacker, prepare_src,
```

Comment:
The removed ones does not exist in the package.

Reply:
?? I'm pretty sure `ChangeDirectory` does, as I think I have a package that uses it.